app/models/green_lanes/measure.rb

```diff
@@ -7,7 +7,7 @@ class Measure < Sequel::Model(:green_lanes_measures)
     many_to_one :goods_nomenclature, class: 'GoodsNomenclature',
                                      primary_key: %i[goods_nomenclature_item_id producline_suffix],
                                      key: %i[goods_nomenclature_item_id productline_suffix]
-    delegate :goods_nomenclature_sid, to: :goods_nomenclature
+    delegate :goods_nomenclature_sid, to: :goods_nomenclature, allow_nil: true
```

Comment:
How this could be nullable as goods_nomenclature_sid is primary key of the [goods_nomenclature] and part of the composite key of green_lanes_measure

Reply:
Its for when the goods nomenclature is nil but this is more thoroughly handled in another PR so I'll just drop this from here